File: dovizKur.cs
using System;
using System.Net;
using System.IO;
using System.Text.Json;

class Program
{
    static void Main()
    {
        dynamic exchangeRates = GetExchangeRates();

        Console.WriteLine("Döviz Kurları ve Altın Fiyatları");
        Console.WriteLine("================================");

        Console.WriteLine("{0,-10} {1,-10} {2,-10} {3,-10}", "Birim", "Satış", "Alış", "Değişim");
        Console.WriteLine("------------------------------------------------------");

        foreach (var rate in exchangeRates)
        {
            string currency = rate.Name;
            decimal satis = rate.satis;
            decimal alis = rate.alis;
            decimal degisim = rate.degisim;

            Console.WriteLine("{0,-10} {1,-10} {2,-10} {3,-10}", currency, satis, alis, degisim);
        }
    }

    static dynamic GetExchangeRates()
    {
        string api_url = "https://api.genelpara.com/embed/doviz.json";
        string data = string.Empty;

        using (WebClient client = new WebClient())
        {
            data = client.DownloadString(api_url);
        }

        dynamic exchangeRates = JsonSerializer.Deserialize<dynamic>(data);
        return exchangeRates;
    }
}

File: dovizKur.php
/**
 * dovizKur.php
 *
 * Bu dosya, genelpara.com API'sinden döviz kurları ve altın fiyatlarını alarak gösterir.
 *
 * Coder: [enesehs]
 * E-posta: [enes.ehs@protonmail.com]
 * Tarih: [T02.08.2023]
 * Sürüm: [V.1]
 *


<!DOCTYPE html>
<html lang="tr" dir="ltr">
<head>
    <meta charset="utf-8">
    <title>Kur Gösterimi</title>
</head>
<body>
<?php
function getExchangeRates() {
    $api_url = "https://api.genelpara.com/embed/doviz.json";
    $data = file_get_contents($api_url);
    return json_decode($data);
}

$exchange_rates = getExchangeRates();
?>

<h1>Döviz Kurları ve Altın Fiyatları</h1>
<table>
    <thead>
        <th>Birim</th>
        <th>Satış</th>
        <th>Alış</th>
        <th>Değişim</th>
    </thead>
    <tbody>
        <?php foreach ($exchange_rates as $currency => $rate) : ?>
            <tr>
                <td><?php echo $currency ?></td>
                <td><?php echo $rate->satis ?></td>
                <td><?php echo $rate->alis ?></td>
                <td><?php echo $rate->degisim ?></td>
            </tr>
        <?php endforeach; ?>
    </tbody>
</table>
</body>
</html>
